### PR TITLE
sniffglue 0.10.1 (new formula)

### PR DESCRIPTION
**Pull request by kpcyrd**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/sniffglue.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Sniffglue < Formula
+  desc "Secure multithreaded packet sniffer"
+  homepage "https://github.com/kpcyrd/sniffglue"
+  url "https://github.com/kpcyrd/sniffglue/archive/v0.10.1.tar.gz"
+  sha256 "27b9448a317957eface67fab1a264057b03b614b588031e4f47655a8e369ead7"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
```
Comment (SMillerDev) [inline Formula/sniffglue.rb]: Please use `--locked` as well.

File: Formula/sniffglue.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    etc.install "sniffglue.conf"
+    man1.install "docs/sniffglue.1"
+  end
+
+  test do
+    system "curl", "-LO", "https://github.com/kpcyrd/sniffglue/raw/master/pcaps/SkypeIRC.pcap"
```
Comment (SMillerDev) [inline Formula/sniffglue.rb]: Please use a resource for this. You can check with other formula how to.

File: Formula/sniffglue.rb (hunk 1 above)
Comment (kpcyrd) [reply Formula/sniffglue.rb]: Done

File: Formula/sniffglue.rb (hunk 2 above)
Comment (kpcyrd) [reply Formula/sniffglue.rb]: I think I got it to work, can you doublecheck if this is the correct way?

File: Formula/sniffglue.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+class Sniffglue < Formula
+  desc "Secure multithreaded packet sniffer"
+  homepage "https://github.com/kpcyrd/sniffglue"
+  url "https://github.com/kpcyrd/sniffglue/archive/v0.10.1.tar.gz"
+  sha256 "27b9448a317957eface67fab1a264057b03b614b588031e4f47655a8e369ead7"
+
+  depends_on "rust" => :build
+
+  resource "pcap" do
```
Comment (SMillerDev) [inline Formula/sniffglue.rb]: Might be better to rename this to `testdata`

File: Formula/sniffglue.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+
+    etc.install "sniffglue.conf"
+    man1.install "docs/sniffglue.1"
+  end
+
+  test do
+    resource("pcap").stage {
```
Comment (SMillerDev) [inline Formula/sniffglue.rb]: `testpath.install resource("testdata")` is probably better. Check `abyss.rb` for usage of resources in the test.

Comment (kpcyrd) [reply Formula/sniffglue.rb]: Thanks, done!

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thank you @kpcyrd for your contribution to Homebrew!

**Pull request closed without merging**